synapse/models/orgs.py

```diff
@@ -240,6 +251,38 @@ def getModelDefs(self):
                     }),
                 )),
                 ('ou:name', {}, ()),
+                ('ou:contract', {}, (
+                    ('title', ('str', {}), {
+                        'doc': 'A terse title for the contract.'}),
+                    ('type', ('str', {'lower': True, 'strip': True}), {
+                        'doc': 'A user-specified type for the contract.'}),
+                    ('sponsor', ('ps:contact', {}), {
+                        'doc': 'The contract sponsor.'}),
+                    ('parties', ('array', {'type': 'ps:contact', 'uniq': True}), {
+                        'doc': 'The non-sponsor entities bound by the contract.'}),
+                    ('document', ('file:bytes', {}), {
+                        'doc': 'The best/current contract document.'}),
+                    ('signed', ('time', {}), {
+                        'doc': 'The date that the contract signing was complete.'}),
+                    ('begins', ('time', {}), {
+                        'doc': 'The date that the contract goes into effect.'}),
+                    ('expires', ('time', {}), {
```

Comment:
In addition to an expiration captures when the contract contractually ends (i think). However do we want to capture the date that contract actually ended  (for example if a contractor was removed from the contract earlier than the expiration date).

Reply:
Maybe something like a `:cancelled` time that the contract was cut short?